content/hacking-atom/sections/hacking-on-atom-core.md

```diff
@@ -83,33 +83,36 @@ In order to build Atom from source, you need to have a number of other requireme
 ##### Requirements
 
 * macOS 10.9 or later
-* Node.js 6.x or later (we recommend installing it via [nvm](https://github.com/creationix/nvm))
-* npm 3.10.x or later (run `npm install -g npm`)
-* Python v2.7.x
+* Node.js 10.12 or later (we recommend installing it via [nvm](https://github.com/creationix/nvm))
+* npm 6 or later (run `npm install -g npm`)
+* Python v2.6.x, v2.7.x or v3.5+
 * Command Line Tools for [Xcode](https://developer.apple.com/xcode/downloads/) (run `xcode-select --install` to install)
 
 {{/mac}}
 
 {{#windows}}
 
-* Node.js 6.9.4 or later (the architecture of node available to the build system will determine whether you build 32-bit or 64-bit Atom)
-* Python v2.7.x
-  * The `python.exe` must be available at `%SystemDrive%\Python27\python.exe`. If it is installed elsewhere create a symbolic link to the directory containing the `python.exe` using: `mklink /d %SystemDrive%\Python27 D:\elsewhere\Python27`
+* Node.js 10.12 or later (the architecture of node available to the build system will determine whether you build 32-bit or 64-bit Atom)
+* Python v2.6.x, v2.7.x, or v3.5+
+  * [Get Python from the Microsoft Store](https://www.microsoft.com/en-us/search/shop/apps?q=python+software+foundation&devicetype=pc&category=Developer+tools%5cDevelopment+kits&Price=0&MaturityRating=ESRB%3aE), or
+  * Download Python from https://www.python.org/downloads/.
+    * For Python 2, be sure to install in the default location, or check "Add Python 2.x to PATH" before installing.
+    * For Python 3, check "Add Python 3.x to PATH", or change the install path to `[Your_Drive_Letter]:\Python37` e.g. `C:\Python37`, (even if your version of Python 3 isn't 3.7, that's one place where the scripts will look.) 
+    * If python isn't found by the bootstrap script, create a symbolic link to the directory containing `python.exe` using e.g.: `mklink /d %SystemDrive%\Python27 D:\elsewhere\Python27`(Links should be set at either `%SystemDrive%\Python27` or `%SystemDrive%\Python37`, regardless of what version of Python you actually have.)
 * 7zip (7z.exe available from the command line) - for creating distribution zip files
```

Comment:
7-zip is a dependency in `scripts/package.json` now, as of https://github.com/atom/atom/pull/17538.

7-Zip as a system installed program wouldn't be used. So it doesn't need to be installed anymore before bootstrapping/building on Windows.

- https://github.com/atom/atom/pull/17538/files#diff-0f080e84d61cfb6f244618e43d478535R5
- https://github.com/atom/atom/pull/17538/files#diff-a2da2402b4e11152e22400e2eb7f52e8R6
- https://github.com/atom/atom/pull/17538/files#diff-a2da2402b4e11152e22400e2eb7f52e8L47-R48